lms/templates/help_modal.html

```diff
@@ -15,7 +15,7 @@
 % if settings.FEATURES.get('ENABLE_FEEDBACK_SUBMISSION', False):
 
 <div class="help-tab">
-  <a href="#help-modal" rel="leanModal" role="button">${_("Help")}</a>
+  <a href="#help-modal" rel="leanModal" role="button">${_("Feedback")}</a>
```

Comment:
Since this is "doing something" rather than "going somewhere" we want to use a button, instead of a link. You may need to hinge additional JavaScript listeners off the new button if the Lean Modal doesn't pick it up.